terraform/region/kms.tf

```diff
@@ -1,3 +1,53 @@
+resource "aws_kms_key" "multi_region_pdf_sqs_encryption_key" {
+  enable_key_rotation = true
+  multi_region        = true
+  provider            = aws.eu-west-1
+}
+
+resource "aws_kms_alias" "multi_region_pdf_sqs_encryption_alias" {
+  name          = "alias/mrk_pdf_sqs_encryption_key-${terraform.workspace}"
+  target_key_id = aws_kms_key.multi_region_pdf_sqs_encryption_key.key_id
+  provider      = aws.eu-west-1
+}
+
+resource "aws_kms_replica_key" "multi_region_pdf_sqs_encryption_key_replica" {
+  description             = "PDF SQS encryption replica key"
```

Comment:
Do replicas automatically rotate with the primary?

Reply:
https://docs.aws.amazon.com/kms/latest/developerguide/multi-region-keys-manage.html#multi-region-rotate yep they do :)